test/socket.router.js

```diff
@@ -42,15 +44,15 @@ describe('socket.router', function(){
 
```

Comment:
It's true, only one test (throwing or emitting) should be required, but this doesn't really seem like a big deal to me to be honest. This redundancy won't hurt anyone.